compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -6902,6 +6902,20 @@ bool OMR::Power::CodeGenerator::inlineDirectCall(TR::Node *node, TR::Register *&
          isGetThenUpdate = true;
          doInline = true;
          }
+      else if (comp->getSymRefTab()->isNonHelper(symRef, TR::SymbolReferenceTable::atomicFetchAndAdd8BitSymbol))
```

Comment:
As this is enabling support for `atomicFetchAndAdd{8|16}BitSymbol` non-helpers on Power, is there any reason why the handling of `atomicFetchAndAddSymbol` shouldn't be revised to allow for 8 and 16 bit operands as well?

Reply:
Similarly, if Power can handle 8 and 16 bit versions of atomic fetch-and-add, is there any reason it can't handle 8 and 16 bit versions of atomic swap operations, below?